accounts-journey/src/main/kotlin/com/backbase/accounts_journey/data/usecase/AccountDetailUseCase.kt

```diff
@@ -1,10 +1,10 @@
 package com.backbase.accounts_journey.data.usecase
 
-import com.backbase.accounts_journey.domain.model.account_detail.AccountDetail
+import com.backbase.android.client.gen2.arrangementclient2.model.AccountArrangementItem
 import dev.drewhamilton.poko.Poko
 
 interface AccountDetailUseCase {
-    suspend fun getAccountDetail(params: Params): Result<AccountDetail>
+    suspend fun getAccountDetail(params: Params): Result<AccountArrangementItem>
```

Comment:
I am still not convinced about exposing client objects directly with the journey's APIs. We risk unnecessarily coupling the client API with journey's API. Clients are generated and inherently produce unstable(especially from past experience) API that strongly tied to the BE. I am not sure it's prudent to strongly couple the 2.